OWNERS_ALIASES

```diff
@@ -0,0 +1,11 @@
+# See the OWNERS_ALIASES docs: https://git.k8s.io/community/contributors/guide/owners.md#OWNERS_ALIASES
+
+aliases:
+  update-approvers:
+    - abhinavdahiya
+    - crawford
+    - jottofar
+    - sdodson
+    - smarterclayton
```

Comment:
Can I be included or I need to be in CVO list first.

Reply:
Done with 2abda3af3 -> 38ca22dac.